src/test/libcephfs/notification_endpoint_api.cc

```diff
@@ -0,0 +1,17 @@
+
+#include "gtest/gtest.h"
+#include "include/compat.h"
+#include "include/cephfs/libcephfs.h"
+#include "include/fs_types.h"
+#include <errno.h>
+#include <fcntl.h>
+
+TEST(LibCephFS, AddUDPEndpoint) {
```

Comment:
Nice to have indeed. Please cover the remaining API calls, I think it should be trivial...